pe-terminal/deb/debian/changelog

```diff
@@ -0,0 +1,5 @@
+pe-terminal (0.0.1-1) unstable; urgency=medium
```

Comment:
Note we'll want this version string here to match the repo tag when we have one.